### DISCUSSION TITLE
just to understand what this stand for

### DISCUSSION THREAD

**Discussion by To-Slalom** [puppylinux-woof-CE/woof-CE#3865] [Q&A]
Hello everyone im trying to make my fossa with all my fav apps, updates since last fossa version come out,... and im making a script to make buitin_files and realize  that you guys have this 3 paths and i wanna understand a bit more about it.

builtin_files          apps that are in puppy_*.sfs
builtin_files-sml    apps that are in advr_*.sfs
builtin_files-norm  but this one i got no clue so if anyone of you can help me ? :) 

P.S. Im working with KuLuSz and trying to learn how to creat scripts,...  and i hope soon to be able to share some of my projects  i will try to share here and Puppy forum too till then best of luck to all and,... 

Best Regards AntonioPt

**Comment by lakshayrohila** [comment]
https://forum.puppylinux.com/viewtopic.php?t=5818 might help you.

**Reply by To-Slalom** [reply]
Thankyou for your replay
